### ISSUE TITLE
Valid email marked as invalid

### ISSUE THREAD

**Issue by jkhulme** [afair/email_address#54]
Hi,

We recently had a user's email get marked as invalid by this Gem, but we are pretty sure it is actually valid as they have emailed us from it.

I don't want to give the users email out, but it was an `<account>@live.co.uk` email address.  And the account was a 2 digit number, followed by 3 letters.

Weirdly there were no errors, it was just being marked invalid.

```
irb(main):047:0> EmailAddress.valid?(email)
=> false
irb(main):048:0> EmailAddress.error(email)
=> ""
```

We're using version 0.1.12 of the gem

**Comment by AlexWayfer** [comment]
```
> gem i email_address
Fetching email_address-0.1.12.gem
Successfully installed email_address-0.1.12
Parsing documentation for email_address-0.1.12
Installing ri documentation for email_address-0.1.12
Done installing documentation for email_address after 0 seconds
1 gem installed

> pry
pry: main > require 'email_address'
=> true

pry: main > email = 'foo@live.co.uk'; puts EmailAddress.valid?(email); puts EmailAddress.error(email)
true

=> nil

pry: main > puts EmailAddress::VERSION
0.1.12
=> nil
```

**Comment by jberry** [comment]
I can, looks like you missed the part about starting with 2 digits:

```
[1] pry(main)> require 'email_address'
=> true

[2] pry(main)> email = '12foo@live.co.uk';

[3] pry(main)> EmailAddress.valid?(email)
=> false

[4] pry(main)> puts EmailAddress::VERSION
0.1.12

```

**Comment by AlexWayfer** [comment]
Sorry, that's true.

But there are errors:

```ruby
pry: main > email = '12foo@live.co.uk'; puts EmailAddress.valid?(email); puts EmailAddress.error(email)
false
Recipient is not valid
```

And some investigation brings me to the https://github.com/afair/email_address/blame/1660263/lib/email_address/config.rb#L146, and the change is https://github.com/afair/email_address/commit/10fc1d30b641378e2e8d72116cdc0c5f77f4d423

So, I guess, it should be closed until there will be other proofs of allowing digits as first characters for MSN.

**Comment by afair** [comment]
Thanks for bringing this forward. It is a bug. My previous research showed that _new_ email addresses must start with a letter, and that only outlook/hotmail domains are allowed anymore. So that is how I set up the check. 

I have a large database of email addresses (at work), and I did find some valid live.com addresses that start with a digit, so it looks like this the rule changed over at the Microsoft/Hotmail/MSN/Live/Outlook identity service. (Fun fact, when created in 1996, Hotmail was branded HoTMaiL to highlight its HTML interface, then sold to MS the next year.)
